enterprise/models.py

```diff
@@ -77,6 +77,35 @@ class Meta(object):
     history = HistoricalRecords()
     identity_provider = models.SlugField(null=True, blank=True, default=None)
```

Comment:
@bradenmacdonald, I'd rather get the existing work up to spec first, and then we can go from there, if that's all right.